_posts/en/newsletters/2024-04-10-newsletter.md

```diff
@@ -0,0 +1,278 @@
+  receiving a forwarding request from Alice or Bob.  Alice and Bob can
+  set outbound fees for their channels to Carol, and it's up to them to
+  set fees that help keep the channels liquid.  Similarly, Carol can
+  adjust her fees for outbound payments to Alice and Bob (e.g. `Dan ->
+  Carol -> Bob`) to help manage liquidity.
+
+  However, Carol may want more control over policies that affect her.
+  For example, if Alice's node is poorly managed, she might frequently
+  forward payments to Carol without many people later wanting to forward
+  payments from Carol to Alice.  That would eventually end with all the
+  funds in their channel ending up on Carol's side, preventing further
+  payments in that direction.  Before this PR, there was nothing Carol
+  could do about that, except close her channel with Alice before it
+  wasted too much of the value of Carol's capital.
+
+  With this PR, Carol can now also charge an _inbound forwarding fee_
+  that is specific to each channel.  For example, she might charge a
+  high fee for payments arriving inbound from Alice's problematic node
+  but a lower fee for payments arriving inbound from Bob's highly liquid
+  node.  Initial inbound fees are expected to always be negative to make
+  them backwards compatible with older nodes that don't understand
+  inbound fees; for example, Carol might give a 10% fee discount on
+  payments forwarded by Bob and a 0% discount on payments forwarded by
+  Alice.
+
+  The fees are assessed simultaneously with the outbound fees.  For
+  example, when Alice offers a payment to Carol for forwarding to Dan,
+  Carol calculates the original `dan_outbound` fee, calculates the new
+  `alice_inbound` fee, and ensures the forwarded payment offers her at
+  least the sum of both.  Otherwise, she rejects the [HTLC][topic htlc].
+  Since the initial inbound fees are always expected to be negative,
+  Carol won't reject any payments that pay sufficient outbound fees, but
+  any node that's now aware of inbound fees may be able to receive a
+  discount.
+
+  Inbound routing fees was first [proposed][bolts #835] about three
+  years ago, was [discussed][jager inbound] on the Lightning-Dev mailing
+  list about two years ago, and was documented in draft [BLIPs #18][]
+  also about two years ago.  Since it's initial proposal, several
+  maintainers of LN implementations besides LND have opposed it.  Some
+  have opposed it on [principle][teinturier bolts835]; others have
+  opposed its design as being [overly specific to LND][corallo overly
+  specific] rather than a local and generic upgrade that can immediately
+  use positive inbound forwarding fees and doesn't require global
+  advertisement of additional fee details for each channel.  An
+  alternative approach is proposed in draft [BLIPs #22][].  We're only
+  aware of one maintainer of a non-LND implementation
+  [indicating][corallo free money] that they'll adopt LND's method---and
```

Comment:
```suggestion
  [indicating][corallo free money] that said they'll adopt LND's method---and
```

Reply:
Leaving as-is.  I think "said" doesn't make sense in this sentence that already contains "indicating".